### PR TITLE
Introduced openssl package.

### PR DESCRIPTION
**Pull request by akornatskyy**
D2IQ-65963

### CODE REVIEW

**Comment by akornatskyy** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by akornatskyy** [comment]
@mesosphere-mergebot request-review

**Comment by akornatskyy** [comment]
@mesosphere-mergebot request-review @jongiddy @DominikDary

File: packages/windows.treeinfo.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "variants": {
+    "openssl",
```
Comment (jongiddy) [inline packages/windows.treeinfo.json]: This needs to indicate that it uses the `windows` variant as with the other entries.

File: packages/openssl/windows.buildinfo.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "sources": {
+    "openssl": {
+      "kind": "url",
+      "url": "https://slproweb.com/download/Win64OpenSSL-1_1_1e.exe",
+      "sha1": "b0b2fa45fd4e4ab6b502495bd2d706b570196911"
```
Comment (jongiddy) [inline packages/openssl/windows.buildinfo.json]: Please use the `downloads.mesosphere` link from https://github.com/mesosphere/dcos-enterprise/blob/345157bc333e0ded07f8938f1d5c1cad83d99994/packages/mesos/windows-ee.buildinfo.json#L21-L25

File: packages/windows.treeinfo.json (hunk 1 above)
Comment (akornatskyy) [reply packages/windows.treeinfo.json]: ah... right...

Comment (akornatskyy) [reply packages/windows.treeinfo.json]: fixed

File: packages/openssl/windows.buildinfo.json (hunk 2 above)
Comment (akornatskyy) [reply packages/openssl/windows.buildinfo.json]: updated

**Comment by DominikDary** [comment]
@mesosphere-mergebot bump-ee

**Comment by DominikDary** [comment]
@mesosphere-mergebot changelog-not-required internal change

**Comment by jongiddy** [comment]
@akornatskyy Can you please add D2IQ-65963 to the initial comment so that this PR gets linked back to the JIRA issue.

**Comment by akornatskyy** [comment]
> @akornatskyy Can you please add D2IQ-65963 to the initial comment so that this PR gets linked back to the JIRA issue.

do you mean description of this PR?

**Comment by jongiddy** [comment]
We usually have a brief description of what the PR is changing, and a reference to the JIRA issue. The code as you have added is enough to trigger the correct linking between JIRA and PR.

**Pull request merged**